src/background/voprf.js

```diff
@@ -9,15 +9,17 @@ import 'asn1-parser';
 
 import sjcl from 'sjcl';
 
+import createKeccakHash from 'keccak';
```

Comment:
consider using [`sha3`](https://www.npmjs.com/package/sha3), which does not rely on Node `Buffer` library which needs to be polyfilled.

Reply:
the keccak package exposes the `squeeze` function that is used as a prng. sha3 does not, and even if i expose the squeeze function of that package and call it multiple times it just return the same hash 🤔 
could you suggest an implementation of that package please 😃

Reply:
let's continue using keccak.